index.html

```diff
@@ -0,0 +1,322 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio-Jorum Odiemo</title>
+    <link rel="stylesheet" href="styles.css">
+</head> 
+<body>
+    <header class="Top">
+        <a href=""><img src="images/MyLogo.png" alt="My Logo" ></a>
+        <nav class="navbar">            
+            <ul class="navbarmenuul">
+                <li class="navbarmenuli">
+                  <a href="#works" class="navbarmenulink">Portfolio</a>
+                </li>
+                <li class="navbarmenuli">
+                  <a href="#about" class="navbarmenulink">About</a>
+                </li>
+                <li class="navbarmenuli">
+                  <a href="#contact" class="navbarmenulink">Contact</a>
+                </li>
+              </ul>
+            </nav> 
+            <nav class="mobile-version">
+                <img class="menu" src="images/menu.png" alt="Menu">        
+            </nav>       
+    </header>
+    <main>
+        <section class="works">
+         <div class="headline">
+            <div class="headline-box">
+              <h2 class="myself">
+                  I’m Jorum<br/>Glad to see you!</h2>
+                  
+              <p class="details">
+                I’m a software developer! I can help you build a product , feature
+                or website Look through some of my work and experience! If you like
+                what you see and have a project you need coded, don’t hestiate to
+                contact me.
+              </p>
+              <p class="social-link-heading">LET’S CONNECT</p>
+              <div class="sociol-logo">
+                <ul class="connect-logo">
+                    <li>
+                        <a href="#"><img src="images/twitterIcon.svg" alt=""></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="images/linkedInIcon.png" alt=""></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="images/mIcon.svg" alt=""></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="images/gitHubIcon.svg" alt=""></a>
+                    </li>
```

Comment:
- Please implement the changes required as stated previously for the desktop view. try to make your design look similar to that of the Figma design [template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)
- 
Please feel free to reach me on slack for more explanation if you find this feedback confusing `@kanu mike chibundu`

### Desktop view
- Please work on the `header section` to properly align the elements in it so that it looks like the Figma design
- Please on the `project section` set a width for each project card and add padding to the main project container. so as to give the spaces on both left and right sides. and also add margin-bottom or gaps in between each project, so they have spaces between them as in the Figma design.
- Please add `margin-top` on the `about section` so it has spaces between the `project section` and the `about section`.
- Please add the `overflow-x: hidden` from your design as your web page scrolls horizontally (left and right) which is not needed for this project.
- [Optional] Please on the contact page, please use the proper placement of the background image, as it is on the Figma design. (This is optional, so it won't affect your project not being approved, but I suggest you add it when you want to, it will make your design more presentable and professional).

##### Your design 
https://user-images.githubusercontent.com/58771507/173183321-63ad43a1-ee1d-4762-9454-e7288752c7b6.mp4


##### Figma design
https://user-images.githubusercontent.com/58771507/173183244-ba08784b-1d89-4685-8b5b-17cba25a272a.mp4